website/src/content/deployment/runtime-configuration.mdx

```diff
@@ -53,38 +63,79 @@ An example configuration for local development:
 }
 ```
 
-# `csp.json`
+
+## Production environment
 
 <Warning>
 
-This file has been deprecated by the `headers.json` mentioned below.
+When the application is deployed on a production environment, the application will fail to make requests to the [API Gateway](/main-concepts/api-gateway). This is due to the fact that the domain, where the application is hosted, is not whitelisted by the CORS rules of the API Gateway.
+
+Therefore, it's important that the application is [registered](/register-applications/configuring-a-custom-application), runs behind the proxy router, and that the `servedByProxy` option in the `env.json` is set to `true`.
 
 </Warning>
 
-To migrate to the new format, wrap the `csp.json` content into a `csp` property in the `headers.json`:
+It is recommended to have two separate configuration files, one for development and one for production. The `env.json` **must** be used for development, therefore for production you can have a `env.prod.json` file.
+
+# `headers.json`
+
+This configuration file allows to configure HTTP headers. The most important one is the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) header.
```

Comment:
```suggestion
This configuration file allows you to configure HTTP headers. The most important one is the [Content Security Policy](https://developer.mozilla.org/en-US/docs/Web/HTTP/CSP) header.
```

Reply:
@adinakleine  the you is not necessary and I am relatively sure that it's not our typical style until now, too.